### PR TITLE
KEYCLOAK-9372 update EAP version

### PR DESCRIPTION
**Pull request by matthewhelmke**

### CODE REVIEW

**Comment by vmuzikar** [comment]
Thanks for the update, @matthewhelmke! Unfortunately, there's more. EAP 7.0 and 7.1 is still mentioned throughout the Securing Applications and Services Guide. Please see e.g. https://github.com/keycloak/keycloak-documentation/blob/master/securing_apps/topics/oidc/java/jboss-adapter.adoc

**Comment by matthewhelmke** [comment]
@vmuzikar There were two files I intentionally left the mentions in.

1. /securing_apps/topics/oidc/java/jboss-adapter.adoc
2. /securing_apps/topics/oidc/java/jboss-adapter.adoc

In both, the mentions are within install instructions on a variety of system configurations. Are we not supporting all those configurations any more?

**Comment by vmuzikar** [comment]
@matthewhelmke I believe, currently is supported 6.4 and 7.2 (and possibly 7.1)? Correct, @pdrozd?

**Comment by matthewhelmke** [comment]
These are the sections in question.

From `securing_apps/topics/saml/java/jboss_adapter_installation.adoc`:

> ifeval::[{project_product}==true]
> .JBoss EAP 7.1 or newer
> [source]
> ----
> $ cd $JBOSS_HOME
> $ ./bin/jboss-cli.sh -c --file=bin/adapter-elytron-install-saml.cli
> ----
> 
> .JBoss EAP 7.0 and EAP 6
> [source]
> ----
> 
> $ cd $JBOSS_HOME
> $ ./bin/boss-cli.sh -c --file=bin/adapter-install-saml.cli
> ----
> 
> NOTE: It is possible to use the legacy non-Elytron adapter on JBoss EAP 7.1 or newer as well, meaning you can use `adapter-install-saml.cli`
> even on those versions. However, we recommend to use the newer Elytron adapter.
> 
> endif::[]

From  `securing_apps/topics/oidc/java/jboss-adapter.adoc`:

> ifeval::[{project_product}==true]
> .JBoss EAP 7.1 or newer
> [source]
> ----
> $ ./bin/jboss-cli.sh --file=bin/adapter-elytron-install-offline.cli
> ----
> 
> .JBoss EAP 7.0
> [source]
> ----
> $ ./bin/jboss-cli.sh --file=bin/adapter-install-offline.cli
> ----
> 
> NOTE: It is possible to use the legacy non-Elytron adapter on JBoss EAP 7.1 or newer as well, meaning you can use `adapter-install-offline.cli`
> even on those versions. However, we recommend to use the newer Elytron adapter.
> 
> NOTE: The offline script is not available for JBoss EAP 6.4
> 
> endif::[]
>

**Comment by pdrozd** [comment]
@vmuzikar @matthewhelmke Correct, EAP 6.4 and 7.2 is supported. EAP 7.1 is also tested and possibly can be used by customers for some time before they move to 7.2. But I would specify only those two versions 6.4 and 7.2

**Comment by matthewhelmke** [comment]
Okay, I'll commit an update shortly.

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 1]
```diff
@@ -126,21 +126,12 @@ even on those versions. However, we recommend to use the newer Elytron adapter.
...
 $ ./bin/jboss-cli.sh --file=bin/adapter-elytron-install-offline.cli
 ----
 
-.JBoss EAP 7.0
-[source]
-----
-$ ./bin/jboss-cli.sh --file=bin/adapter-install-offline.cli
-----
-
-NOTE: It is possible to use the legacy non-Elytron adapter on JBoss EAP 7.1 or newer as well, meaning you can use `adapter-install-offline.cli`
```
Comment (vmuzikar) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc:141]: I believe the note should remain there.

**Review by vmuzikar — COMMENTED** [review]
@matthewhelmke Thanks!

Just a few notes.

EAP 7.0 is also mentioned here: https://github.com/keycloak/keycloak-documentation/blob/master/securing_apps/topics/saml/java/jboss-adapter/jboss-adapter-rpms.adoc

**Comment by matthewhelmke** [comment]
Thank you. Both will be fixed in the next commit, coming momentarily.

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 2]
```diff
@@ -126,21 +126,12 @@ even on those versions. However, we recommend to use the newer Elytron adapter.
 endif::[]
 
 ifeval::[{project_product}==true]
-.JBoss EAP 7.1 or newer
+.JBoss EAP 7.2 or newer
```
Comment (stianst) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: This should be 7.1 or newer - installation steps changed in 7.1 (not 7.2)

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 3]
```diff
@@ -162,13 +153,15 @@ $ ./bin/jboss-cli.sh -c --file=bin/adapter-install.cli
...
```
Comment (stianst) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: This should be 7.1 or newer - installation steps changed in 7.1 (not 7.2)

File: securing_apps/topics/oidc/java/jboss-adapter.adoc (hunk 1 above)
Comment (stianst) [reply securing_apps/topics/oidc/java/jboss-adapter.adoc:141]: Yes, this should remain

Comment (stianst) [reply securing_apps/topics/oidc/java/jboss-adapter.adoc:141]: This file should just be left unchanged

File: securing_apps/topics/saml/java/jboss-adapter/jboss_adapter_installation.adoc [hunk 4]
```diff
@@ -76,22 +76,22 @@ even on those versions. However, we recommend to use the newer Elytron adapter.
 endif::[]
 
 ifeval::[{project_product}==true]
-.JBoss EAP 7.1 or newer
+.JBoss EAP 7.2 or newer
```
Comment (stianst) [inline securing_apps/topics/saml/java/jboss-adapter/jboss_adapter_installation.adoc]: This should be 7.1 or newer - installation steps changed in 7.1 (not 7.2)

File: securing_apps/topics/saml/java/jboss-adapter/jboss_adapter_installation.adoc [hunk 5]
```diff
@@ -76,22 +76,22 @@ even on those versions. However, we recommend to use the newer Elytron adapter.
...
 [source]
 ----
 $ cd $JBOSS_HOME
 $ ./bin/jboss-cli.sh -c --file=bin/adapter-elytron-install-saml.cli
 ----
 
-.JBoss EAP 7.0 and EAP 6
+.JBoss EAP 6.4
```
Comment (stianst) [inline securing_apps/topics/saml/java/jboss-adapter/jboss_adapter_installation.adoc]: Leave as is

File: securing_apps/topics/saml/java/jboss-adapter/jboss_adapter_installation.adoc [hunk 6]
```diff
@@ -76,22 +76,22 @@ even on those versions. However, we recommend to use the newer Elytron adapter.
...
 [source]
 ----
 
 $ cd $JBOSS_HOME
 $ ./bin/boss-cli.sh -c --file=bin/adapter-install-saml.cli
 ----
 
-NOTE: It is possible to use the legacy non-Elytron adapter on JBoss EAP 7.1 or newer as well, meaning you can use `adapter-install-saml.cli`
+NOTE: It is possible to use the legacy non-Elytron adapter on JBoss EAP 7.2 or newer as well, meaning you can use `adapter-install-saml.cli`
```
Comment (stianst) [inline securing_apps/topics/saml/java/jboss-adapter/jboss_adapter_installation.adoc]: Leave as is

File: openshift/topics/get_started.adoc [hunk 7]
```diff
@@ -480,9 +480,9 @@ $ oc new-build --binary=true \
 --name=eap-app
 --> Found image 31895a4 (3 months old) in image stream "openshift/jboss-eap71-openshift" under tag "latest" for "jboss-eap71-openshift"
 
-    JBoss EAP 7.1
+    JBoss EAP 7.2
```
Comment (stianst) [inline openshift/topics/get_started.adoc]: Use template variable for EAP version. {eap_version} or something like that. Applies to changes below as well.

**Review by stianst — CHANGES_REQUESTED** [review]

**Comment by matthewhelmke** [comment]
@stianst Changes and reversions made.

**Review by stianst — APPROVED** [review]

**Pull request merged**